File: Python/File/move_file_from_old_to_new_path.py
import os



# Move a file from a old path to a new path.
def move_file_from_old_to_new_path(old_path, new_path):
    os.rename(old_path, new_path)


def main():
    # Move a file from a old path to a new path.
    move_file_from_old_to_new_path("assets/valid/valid-txt.txt", "assets/valid/valid.txt")



main()

File: Rust/src/File/move_file_from_old_to_new_path.rs
#![allow(clippy::needless_return)]
use std::fs;

fn main() {
    // Move a file from old path to a new path.
    move_file_from_old_to_new_path("assets/valid/valid-txt.txt", "assets/valid/valid.txt");
}

// Move a file from old path to a new path
fn move_file_from_old_to_new_path(old_filepath: &str, new_filepath: &str) {
    fs::rename(old_filepath, new_filepath).unwrap();
}
